src/ssl.c

```diff
@@ -45110,7 +45228,7 @@ int wolfSSL_X509_NAME_print_ex_fp(XFILE file, WOLFSSL_X509_NAME* name,
     return ret;
 }
 #endif /* NO_FILESYSTEM */
-
+#endif /* ----------------------test ----------------------- */
```

Comment:
Looks like a test/debug line got left in here?  Please remove.

Reply:
Removed the line.